pkgs/by-name/to/tocpdf/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "tocpdf";
+  version = "0.3.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "tocPDF";
+    inherit version;
+    hash = "sha256-B+UcvyjWceVErf1uDyGGTGwKBCGHmSOF19Vbk15cPp8=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    wheel
+  ];
+
+  dependencies = with python3Packages; [
+    click
+    pdfplumber
+    pypdf
+    tika
+    tqdm
+  ];
+
+  pythonImportsCheck = [ "tocPDF" ];
+
+  meta = with lib; {
+    description = "Automatic CLI tool for generating outline of PDFs based on the table of contents";
+    homepage = "https://github.com/kszenes/tocPDF";
+    license = licenses.mit;
+    maintainers = with maintainers; [ dansbandit ];
+    mainProgram = "tocPDF";
+  };
```

```suggestion
  meta = {
    description = "Automatic CLI tool for generating outline of PDFs based on the table of contents";
    homepage = "https://github.com/kszenes/tocPDF";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ dansbandit ];
    mainProgram = "tocPDF";
  };
```
could you avoid `with lib` here? It may cause a confusion. refer: https://github.com/NixOS/nixpkgs/issues/208242